<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QORIL8E" failures="1" tests="5" name="hello" time="34.081" errors="0" timestamp="2022-08-24T12:12:10 IST">
  <testcase classname="TestNGDemo.dependsonMethod" name="login" time="0.017">
    <failure type="java.lang.AssertionError" message="expected [nn] but found [name]">
      <![CDATA[java.lang.AssertionError: expected [nn] but found [name]
at TestNGDemo.dependsonMethod.login(dependsonMethod.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="TestNGDemo.dependsonMethod" name="logout" time="0.003"/>
  <testcase classname="TestNGDemo.dependsonMethod" name="application" time="0.0">
    <skipped/>
  </testcase> <!-- application -->
  <testcase classname="TestNGDemo.FlowExection" name="test1" time="3.931"/>
  <testcase classname="TestNGDemo.FlowExection" name="test1" time="3.03"/>
</testsuite> <!-- hello -->
